File: app/src/main/java/com/belous/v/clrc/model/db/AppDatabase.java
package com.belous.v.clrc.model.db;

import androidx.room.Database;
import androidx.room.RoomDatabase;

import com.belous.v.clrc.model.Item;

@Database(entities = {Item.class}, version = 1, exportSchema = false)
public abstract class AppDatabase extends RoomDatabase {
    public static final String DB_NAME = "database";

    public abstract ItemDao itemDao();
}

File: app/src/main/java/com/belous/v/clrc/data/db/AppDatabase.kt
package com.belous.v.clrc.data.db

import androidx.room.Database
import androidx.room.RoomDatabase
import androidx.room.TypeConverters
import com.belous.v.clrc.data.db.entity.YeelightEntity
import com.belous.v.clrc.data.db.entity.ParamsConverter

@Database(entities = [YeelightEntity::class], version = 1)
@TypeConverters(ParamsConverter::class)
abstract class AppDatabase : RoomDatabase() {

    companion object {
        const val DB_NAME = "database"
    }

    abstract fun itemDao(): YeelightDao
}